packages/loaders/git/src/index.ts

```diff
@@ -86,17 +87,9 @@ export class GitLoader implements UniversalLoader {
 
     const rawSDL = gqlPluckFromCodeStringSync(pointer, content, options.pluckConfig);
 
-    return ensureSource({ rawSDL, pointer, path });
-  }
-}
-
-function ensureSource({ rawSDL, pointer, path }: { rawSDL: string; pointer: string; path: string }) {
```

Comment:
Dropping `ensureSource` and `createInvalidExtensionError` because `gqlPluckFromCodeString` guarantees a string is returned and will already throw if an extension is invalid.